### DISCUSSION TITLE
kafka exporter installation problem

### DISCUSSION THREAD

**Discussion by rahulkinge22** [strimzi/strimzi-kafka-operator#5555] [Q&A]
Hello: please help me to add  kafka exporter in strimzi/kafka:0.19.0-kafka-2.5.0 operator.
i am trying to add KafkaEXporter: {} in kafka custom resources, But its not working.

**Comment by scholzj** [comment]
I guess you would need to share more details ... logs, the YAML with the `Kafka` customer resource etc. One thing of note ... it is `kafkaExporter: {}`, not `KafkaEXporter: {}`

**Comment by rahulkinge22** [comment]
[kafka.txt](https://github.com/strimzi/strimzi-kafka-operator/files/7156378/kafka.txt)
 I added kafkaExporter: {} in kafka crd but its not creating exporter pod
can you please guide me where exactly i need  to add this parameter ?

please check the attached kafka crd file as a ref.

**Reply by scholzj** [reply]
You have to distinguish the CR and CRD. CRD is the definition which extends the Kubernetes API. You just apply the files shipped by Strimzi and never modify them. The CR is the instance of the CRD which you create to deploy the Kafka cluster. That is where you configure the Kafka Exporter. Here is an example of the Kafka CR including the exporter configuration which you can just copy: https://github.com/strimzi/strimzi-kafka-operator/blob/0e11dd79b19611ad1cd47f7370299c3069f4f258/examples/metrics/kafka-metrics.yaml#L64 ... the file you shared is the CRD which you are not supposed to modify.
